embed_temp/embed_temp.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+class Embed_Template:
```

Comment:
Please can this be renamed to “EmbedTemplate”. Also move into a folder called static/embed_template.py. Remember to create an __init__ file too and add this class to it. Then add the static folder/module to the __init__ file in the same level as main.py